Automattic/vip-go-mu-plugins
# Update/jetpack 9.7 beta 2

## Description

Updated the subtree for the Jetpack 9.7 release, with Jetpack 9.7-beta2

## Changelog Description

### Added Jetpack 9.7-beta2

We have updated the subtree for Jetpack 9.7, containing version 9.7-beta2. If you want to try this beta, you can follow the [testing guidelines](https://github.com/Automattic/jetpack/blob/jetpack/branch-9.7/to-test.md).

As usual, you need to pin your site to use Jetpack 9.7 by adding this line to your `vip-config.php`:

```
define( 'VIP_JETPACK_PINNED_VERSION', '9.7' );
```

## Checklist


## Steps to Test

1. Check out PR.
2. Pin your testing site to the new subtree version: `define( 'VIP_JETPACK_PINNED_VERSION', '9.7' );`
3. Go to the Jetpack dashboard in `wp-admin` and ensure version 9.7-beta2 is active
4. Ensure everything else works as expected
